cfgov/unprocessed/css/organisms/video-player.less

```diff
@@ -146,6 +146,11 @@
         left: 50%;
         top: 50%;
         transform: translate( -50%, -50% );
+        
+        &:focus-visible {
```

Comment:
This will only apply to the centered play button. We have the `bottom-right` version also (line 156).